node/config/types.go

```diff
@@ -51,6 +51,7 @@ type Boost struct {
 	HttpDownload        HttpDownloadConfig
 	Retrievals          RetrievalConfig
 	IndexProvider       IndexProviderConfig
+	Curio               CurioConfig
```

Comment:
Wouldn't it be better to put this section near the top, after `SectorIndexApiInfo` so that backend configuration in all near the top?